src/components/PlanCard/PlanCard.styled.tsx

```diff
@@ -1,29 +1,38 @@
 import styled from 'styled-components';
-import { BsThreeDotsVertical } from 'react-icons/bs';
+import { BsThreeDotsVertical, BsHeart } from 'react-icons/bs';
 import { Card } from '../Card';
 
 export const CardContainer = styled(Card)`
   position: relative;
   border-radius: 0;
-  background: ${({ theme }) => theme.color.primary};
-  border: 1.75px solid ${({ theme }) => theme.color.grey};
+  background: ${({ theme }) => theme.color.white};
+  border: 1px solid ${({ theme }) => theme.color.primary};
+`;
+export const CardTopNav = styled.section`
+  height: 20%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background: ${({ theme }) => theme.color.white};
 `;
 export const PlanInfoWrapper = styled.section`
+  background: ${({ theme }) => theme.color.primary};
   position: relative;
   width: 100%;
-  height: 75%;
+  height: 50%;
   border-bottom: 1.75px solid ${({ theme }) => theme.color.grey};
 `;
 export const PlanInfo = styled.section`
-  display: flex;
-  justify-content: space-between;
-  align-items: center;
   padding: 15px;
+  align-items: center;
 `;
 export const PlanId = styled.p`
-  color: ${({ theme }) => theme.color.white};
-  font-size: 1.1rem;
-  font-weight: 700;
+  font-size: 1.3rem;
+  font-weight: 600;
+  padding-left: 6px;
```

Comment:
I think we should do em / rem for padding for consistency

Reply:
Done! Thank you.